data/topic-schema/src/main/kotlin/net/corda/schema/Schemas.kt

```diff
@@ -97,6 +97,8 @@ class Schemas {
             const val PROPOSAL_TOPIC = "membership.proposals"
             const val MEMBERSHIP_RPC_TOPIC = "membership.rpc.ops"
             const val MEMBERSHIP_RPC_RESPONSE_TOPIC = "membership.rpc.ops.resp"
+            const val MEMBERSHIP_RPC_CERTIFICATES_TOPIC = "membership.rpc.certificates.ops"
```

Comment:
Similar comment as for the avro schemas. Maybe we shouldn't tie this to membership since it can be cluster level.